terraform/girder.tf

```diff
@@ -0,0 +1,29 @@
+data "local_file" "ssh_public_key" {
+  filename = "girder-dandi-server_id_rsa.pub"
+}
+
+resource "aws_route53_record" "girder" {
+  zone_id = aws_route53_zone.dandi.zone_id
+  name    = "girder"
+  type    = "A"
+  ttl     = "300"
+  records = [module.girder_server.ec2_ip]
+}
+
+module "girder_server" {
+  source  = "girder/girder/aws//modules/server"
+  version = "0.6.0"
+
+  project_slug      = "dandi-girder"
+  ssh_public_key    = data.local_file.ssh_public_key.content
+  ec2_volume_size = 100
+}
+
+module "girder_smtp" {
+  source  = "girder/girder/aws//modules/smtp"
```

Comment:
just curious, is there some added semantic behind `//`?

Reply:
Yes, it's Terraform Registry's syntax for submodules: https://registry.terraform.io/modules/girder/girder/aws/0.7.0/submodules/smtp